tree/ntuple/v7/src/RNTupleMerger.cxx

```diff
@@ -195,7 +198,8 @@ struct RColumnMergeInfo {
    // the columns' parent fields' names plus the index of the column itself.
    // e.g. "Muon.pt.x._0"
    std::string fColumnName;
-   DescriptorId_t fInputId;
+   DescriptorId_t fInputLogicalId;
+   DescriptorId_t fInputPhysicalId;
```

Comment:
I think we may not need at all the logical columns.

Reply:
Hm, I'm confused: Didn't we recently change this that alias columns (ie where logical column ids matter) are always higher than physical columns, which is what the merger probably should work on?

Reply:
Yes, the merger should only work on non-alias columns, so I will remove the distinction and just call it `fInputId`